### PR TITLE
Inline icons. Remove icon install step.

### PR DESCRIPTION
**Pull request by nhunzaker**
In order to make installation easier, I've inlined the SVG icons into the project.
I've also eliminated some pseudo-selectors (I'd like to eventually drop node-sass)

### CODE REVIEW

File: src/icons/search-icon.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+const SearchIcon: React.SFC<Props> = ({ title }) => (
+  <IconFrame>
+    <title>{title}</title>
+    <path
+      opacity="0.54"
+      d="M15.5,14h-0.79l-0.28-0.27C15.41,12.59,16,11.11,16,9.5C16,5.91,13.09,3,9.5,3S3,5.91,3,9.5S5.91,16,9.5,16
+    c1.61,0,3.09-0.59,4.23-1.57L14,14.71v0.79l5,4.99L20.49,19L15.5,14z M9.5,14C7.01,14,5,11.99,5,9.5S7.01,5,9.5,5S14,7.01,14,9.5
+    S11.99,14,9.5,14z"
```
Comment (nhunzaker) [inline src/icons/search-icon.tsx]: Huh. This shouldn't work. Does TypeScript support multiple line strings?

File: src/components/selection.tsx [hunk 2]
```diff
@@ -42,7 +43,7 @@ export default class Selection extends React.Component<Props, {}> {
               resource={resource}
             />
 
-            <span className="ars-selection-button-icon" aria-hidden="true" />
+            <EditIcon />
```
Comment (dbernheisel) [inline src/components/selection.tsx]: Does this remove `aria-hidden=true`? I wonder if this removes some accessibility support.

**Comment by dbernheisel** [comment]
@nhunzaker any chance that we can we could make these icons replaceable components? I anticipate that since the instruction before was to `copy icons to your public directory`, that this gave other devs the ability to customize the icons for their apps. With inlining, it's removes that ability.

File: src/components/selection.tsx (hunk 2 above)
Comment (nhunzaker) [reply src/components/selection.tsx]: Good question. I'm not clear if the empty span causes a verbal change, but it's easy enough to pass through this prop.

Comment (cwmanning) [reply src/components/selection.tsx]: Last I heard, an empty element is not read, making `aria-hidden` redundant in those cases.

**Comment by nhunzaker** [comment]
Moved icons into a single module. Ran out of time, but next I'll make them into an object that can be passed as options.

**Pull request merged**